WordPress/src/main/java/org/wordpress/android/viewmodel/main/WPMainActivityViewModel.kt

```diff
@@ -321,7 +319,7 @@ class WPMainActivityViewModel @Inject constructor(
     }
 
     private fun shouldShowStories(site: SiteModel?): Boolean {
-        return wpStoriesFeatureConfig.isEnabled() && SiteUtils.supportsStoriesFeature(site)
+        return SiteUtils.supportsStoriesFeature(site)
```

Comment:
What do you think about removing this simple method and replacing wherever it's called with `SiteUtils.supportsStoriesFeature(site)` directly?

Reply:
We can leave it if you think it adds clarity - it's just that the occurrences where this is used are already giving reading clarity by assigning the return value to a descriptive vairable i.e. `val shouldShowStories = shouldShowStories(site)`

Reply:
I've implemented your suggestion in a9b2936421, I think it makes sense 👍